src/components/modals/LogsModal.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react'
+import { Platform, View } from 'react-native'
+import { AirshipBridge } from 'react-native-airship'
+import RNFS from 'react-native-fs'
+import Share, { ShareOptions } from 'react-native-share'
+
+import { getLogOutput } from '../../actions/LogActions'
+import s from '../../locales/strings'
+import { useDispatch } from '../../types/reactRedux'
+import { showError } from '../services/AirshipInstance'
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { MainButton } from '../themed/MainButton'
+import { ModalCloseArrow, ModalMessage, ModalTitle } from '../themed/ModalParts'
+import { OutlinedTextInput } from '../themed/OutlinedTextInput'
+import { ThemedModal } from '../themed/ThemedModal'
+
+interface LogsModalProps {
+  bridge: AirshipBridge<void>
+}
+
+export const LogsModal = ({ bridge }: LogsModalProps) => {
+  const theme = useTheme()
+  const dispatch = useDispatch()
+  const styles = getStyles(theme)
+  const [spinning, setSpinning] = React.useState(false)
+  const [userMessage, setUserMessage] = React.useState('')
+  const dir = Platform.OS === 'android' ? RNFS.ExternalCachesDirectoryPath : RNFS.DocumentDirectoryPath
```

Comment:
Put this inside `handleShare`, since it isn't used anywhere else.

Reply:
Moved inside `handleShare`